### PR TITLE
Add e2e test for backup by dumper

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Close  https://github.com/pingcap/tidb-operator/issues/2348

In c1ee9de, dumper e2e case have passed the test.

https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/operator_ghpr_e2e_test_kind/detail/operator_ghpr_e2e_test_kind/6206/pipeline/65

And then I move the dumper e2e case from basic context to the br context.

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by Yisaer** [comment]
Currently dumper Bracker haven't support `Prefix` path, we should wait https://github.com/pingcap/tidb-operator/pull/2435 merged.

File: tests/e2e/tidbcluster/tidbcluster.go [hunk 1]
```diff
@@ -428,31 +484,37 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
...
+			framework.Skipf("CRD:Backup and restore with BR only test in aws/kind")
+		}
+		err := backupCRDTest(provider, ns, fixture.BRType, fw, genericCli, c, cli)
 		framework.ExpectNoError(err)
+	})
 
-		err = storage.CheckDataCleaned()
+	ginkgo.It("CRD:Backup and restore with Dumper", func() {
+		provider := framework.TestContext.Provider
+		if provider != "kind" {
```
Comment (DanielZhangQD) [inline tests/e2e/tidbcluster/tidbcluster.go:513]: Mydumper + Lightning also supports aws.

Comment (Yisaer) [reply tests/e2e/tidbcluster/tidbcluster.go:513]: We will support aws later

**Comment by Yisaer** [comment]
defer cancel function and create 2 cluster first then waitcluster ready.

File: cmd/backup-manager/app/constants/constants.go [hunk 2]
```diff
@@ -64,5 +64,5 @@ const (
 	ServiceAccountCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 
 	// DefaultTableRegex is the default regular expression for 'db.table' matching
-	DefaultTableRegex = "^(?!(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))"
+	DefaultTableRegex = "^(?!(mysql|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))"
```
Comment (Yisaer) [inline cmd/backup-manager/app/constants/constants.go:67]: @onlymellb As our e2e test hard code using `test` to insert data, I suggest remove `test` from DefaultTableRegex. WDYT.

Comment (onlymellb) [reply cmd/backup-manager/app/constants/constants.go:67]: When using mydumper to back up, users do not need to back up the test DB by default, so I think it is better to create a new database in e2e to insert data.

**Comment by Yisaer** [comment]
In c1ee9de, dumper e2e case have passed the test.

https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/operator_ghpr_e2e_test_kind/detail/operator_ghpr_e2e_test_kind/6206/pipeline/65

**Pull request merged**